README.md

```diff
@@ -0,0 +1,50 @@
+[![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
+
+# OpenTracing Apache HttpClient 
+
+Apache HttpClient instrumentation.
+
+## Configuration
+
+```java
+GlogalTracer.register(tracer);
+HttpClient httpClient = new TracingHttpClientBuilder()
```

Comment:
It has limitations, it would not finish/mark as error if request results in an error (e.g. unknown host). Which is a big disadvantage. This approach is going to be in brave4 too.